### ISSUE TITLE
Configured table prefix is not recognized in CLI admin:user:create

### ISSUE THREAD

**Issue by crtl** [magento/magento2#11176]
I have installed Magento via the CLI and then wanted to create a new admin user via the CLI.

### Preconditions

1.  Installed Magento 2.2.0 instance with configured tableprefix

### Steps to reproduce

```
 php bin/magento admin:user:create --admin-user=user --admin-password=pass --admin-email=mail@mail.com --admin-firstname=fname --admin-lastname=lname
```

### Expected result
1.  A new User should be created
2. The tableprefix should be properly prepended to tablenames

### Actual result

1. The user could no be created because the table was not found.

```
  [Zend_Db_Statement_Exception]
  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'my_db.admin_passwords' doesn't exist, query was: INSERT INTO `
  admin_passwords` (`user_id`, `password_hash`, `last_updated`) VALUES (?, ?, ?)


  [PDOException]
  SQLSTATE[42S02]: Base table or view not found: 1146 Table 'my_db.admin_passwords' doesn't exist
```

**Comment by magento-engcom-team** [comment]
@crtl, thank you for your report.
We've created internal ticket(s) MAGETWO-80545 to track progress on the issue.

**Comment by denisristic** [comment]
I'm working on it #SQUASHTOBERFEST

**Comment by denisristic** [comment]
As I can see @osrecio fixed that in #11199

**Comment by okorshenko** [comment]
The issue has been fixed and merged to 2.2-develop branch. Will be available with 2.2.2 release

**Comment by magento-team** [comment]
Internal ticket to track issue progress: MAGETWO-81645

**Comment by magento-engcom-team** [comment]
Hi @crtl. Thank you for your report. 
The issue has been fixed in magento-engcom/magento2ce#1283 by @magento-engcom-team in 2.3-develop branch
Related commit(s):
 - 074285735cd6d33fd530302390ab8101e73d7e34
 - fde512eacfce36508bcec2b1c08ef381d0af3f92
 - e7d90de918be70731e2021dee757d1e7af374ab8
 - 6d4f36db79a9bee553b4acfe8c0820b5c046e6af
 - 2e83a36fbf3201bdaf6612dcfce08125687f0e28
 - 23b4aa11f5a4a0b6bfe76cd1e812d36d88ed4d83
 - b50bfa811876af007426b734019fd149def63bba
 - 04cd60da14a87727768165942c23fc1403016e41
 - 75d5a0522e8a18c2f403ea284d62e445e3d6ddc2
 - 5e02b15fcbe310cd13e385d0ae7a3e6017b56497
 - aad5ba6236ca8557072f62db524b23d68eae5de3
 - 3e321401374a7f2292bd3b2c4540db2c0af34216
 - 7898946b7ec40999b2007ee377d6b6312a3778b3
 - a173e1cd160b1d7cd24a855420317fdd7bfd9624
 - 41d0075104cb628ec1806130b6449929e8585997
 - dda19b2f6eff23e0904673fe7719c3fd935aed02
 - cba48cdfedc500b652deee50e3a3dc95e366eca3
 - 0c62caaf80c9659530d0c3dd0741108d1b0dadb0
 - 736f783b1b2fd361fb8328fd7c5cf9e7c3c172e2
 - 7569b7002878019a115aa2ea587d476a282fcd17

The fix will be available with the upcoming patch release.

**Comment by dmitry-fedyuk** [comment]
My fix for Magento < 2.3: https://mage2.pro/t/5753/3
